plugins/generator-1.21.8/neoforge-1.21.8/procedures/entity_allow_flying.java.ftl

```diff
@@ -1,4 +1,5 @@
-if(${input$entity} instanceof Player _player) {
+<@head>if (${input$entity} instanceof Player _player) {</@head>
     _player.getAbilities().mayfly = ${input$condition};
+<@tail>
     _player.onUpdateAbilities();

```

Comment:
This line is indented using whitespace instead of tab, resuling in non-optimised code in 1.21.8 too:

<img width="387" height="216" alt="image" src="https://github.com/user-attachments/assets/0069eb3b-2f68-49a2-a096-288d803d5fe5" />

Please double check all blocks